File: 백준/Bronze/3040. 백설 공주와 일곱 난쟁이/백설 공주와 일곱 난쟁이.py
res = 0
arr = []
for i in range(9):
    val = int(input())
    arr.append(val)
res = sum(arr)
for j in range(8):
    for k in range(j + 1, 9):
        if res - (arr[j] + arr[k]) == 100:
            a1 = arr[j]
            b1 = arr[k]
            break
arr.remove(a1)
arr.remove(b1)
arr.sort()
for n in arr:
    print(n)

File: 백준/Bronze/3040. 백설 공주와 일곱 난쟁이/백설 공주와 일곱 난쟁이.cc
#include <iostream>

using namespace std;

int main(){
    int arr[9], sum = 0;
    int a, b;
    
    for(int i = 0; i < 9; i++){
        cin >> arr[i];
        sum += arr[i];
    }
    
    for(int i = 0; i < 9; i++){
        for(int j = i + 1; j < 9; j++){
            if(sum - (arr[i] + arr[j]) == 100){
                a = i; 
                b = j;
                break;
            }
        }
    }
    for(int k = 0; k < 9; k++){
        if(k == a || k == b) continue;
        else cout << arr[k] << endl;
    }
}